### PR TITLE
Feature/harrison

### PR DESCRIPTION
**Pull request by Mefisa**
template for express app

### CODE REVIEW

**Comment by fzheng1** [comment]
Can you link the youtube video you used to make this

File: build/app.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+"use strict";
+
+var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");
```
Comment (fzheng1) [inline build/app.js:3]: why underscores?

File: build/app.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+var app = (0, _express["default"])();
+app.use((0, _morgan["default"])('dev'));
+app.use(_express["default"].json());
+app.use(_express["default"].urlencoded({
+  extended: true
+}));
+app.use((0, _cookieParser["default"])());
+app.use('/v1', _index["default"]);
+var _default = app;
+exports["default"] = _default;
```
Comment (fzheng1) [inline build/app.js:27]: add newline

File: build/routes/index.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+var indexRouter = _express["default"].Router();
+
+indexRouter.get('/', function (req, res) {
+  return res.status(200).json({
+    message: 'Welcome to Express API template'
+  });
+});
+var _default = indexRouter;
+exports["default"] = _default;
```
Comment (fzheng1) [inline build/routes/index.js:20]: add newline

File: src/app.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+import cookieParser from 'cookie-parser';
+import indexRouter from './routes/index.js';
+const app = express();
+app.use(logger('dev'));
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+app.use(cookieParser());
+app.use('/v1', indexRouter);
+
+export default app;
```
Comment (fzheng1) [inline src/app.js]: newline

File: src/routes/index.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+
+const indexRouter = express.Router();
+
+indexRouter.get('/', (req, res) =>
+  res.status(200).json({ message: 'Welcome to Express API template' })
+);
+
+export default indexRouter;
```
Comment (fzheng1) [inline src/routes/index.js]: newline

**Review by fzheng1 — APPROVED** [review]
lgtm just add newlines at end of files

File: build/app.js (hunk 1 above)
Comment (Mefisa) [reply build/app.js:3]: this is what babel builds

**Pull request merged**